notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1120_az_brake_analysis.ipynb

```diff
@@ -0,0 +1,1197 @@
+{
```

Comment:
<p>Adding a quick one line description as the following might be helpful:</p><blockquote>A function to actual and demand position and velocity of TMA azimuth with respect to time. It includes vertical lines to indicate two different event and plot for interpolated acceleration. </blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>